Gemfile.lock

```diff
@@ -261,14 +261,14 @@ GEM
       rspec-mocks (~> 3.4.0)
       rspec-support (~> 3.4.0)
     rspec-support (3.4.1)
-    rubocop (0.37.2)
-      parser (>= 2.3.0.4, < 3.0)
+    rubocop (0.40.0)
```

Comment:
(just trying to learn how you all do things) what are the steps that you are following to see the error?